src/textarea/src/Textarea.js

```diff
@@ -26,7 +41,12 @@ const Textarea = memo(
       ...restProps
     } = props
 
-    const { className: themedClassName, boxProps } = useInputAppearance()
+    const { className: themedClassName, ...boxProps } = useStyleConfig(
+      'Input',
```

Comment:
We probably need to call out that these don't match the component names in the theming docs.

Reply:
Or in the theme you could split them up and import the same file for both exports `TextInput` and `Textarea`?

Reply:
Yeah, I definitely want to break these apart.